src/components/UI/InputText.vue

```diff
@@ -1,14 +1,21 @@
 <template>
   <div>
     <input
+      v-focus="isFocus"
       :type="isNumber ? 'number' : 'text'"
-      :class="[$style.input, disabled ? $style.disabled : '']"
+      :class="[
+        $style.input,
+        disabled ? $style.disabled : '',
+        isHover ? $style.hover : ''
+      ]"
       :placeholder="placeholder"
       :disabled="disabled"
       :value="modelValue"
       @input="update"
     />
-    <input-focus-underline :class="$style.underline" />
+    <slot>
+      <input-focus-underline :class="$style.underline" />
+    </slot>
```

Comment:
```suggestion
    <input-focus-underline :class="$style.underline" />
    <slot>
    </slot>
```
こうしたいです(cssの繰り返しを避けたい)